docs/admin/settings.rst

```diff
@@ -170,7 +170,7 @@ Engine settings
   is described in the file.
 
 ``disabled`` : optional
-  To disable by default the engine, but not deleting it. It will allow the user
+  To disable by default the engine, but not deleting it.  It will allow the user
```

Comment:
Same question, why is this extra space needed?

Reply:
see https://github.com/asciimoo/searx/pull/2063#discussion_r453509512